packages/slate/src/utils/get-normalize-setting.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * Obtains the normalization setting to use for the change based on the
+ * change object flags and the change operation options.
+ *
+ * @param {Object} flags - flags object for the current change object
+ * @param {Object} options - options for the change operation
+ * @return {bool} - the normalization configuration to use. Defaults to true
+ *   if flags or options does not have a normalization configuration
+ */
+
+function getNormalizeSetting(flags, options) {
```

Comment:
I think we should change this from being a util to being a method on `Change` objects. So the calling code can change from:

```js
const normalize = getNormalizeSetting(change.flags, options)
```

Into...

```js
const normalize = change.getFlag('normalize', options)
```

(Later we can even change `setOperationFlag` and `unsetOperationFlag` to just be `un/setFlag` to match the terser API too.)

Reply:
Great idea